pylhc/bpm_calibration/calibration.py

```diff
@@ -0,0 +1,180 @@
+"""
+BPM Calibration
```

Comment:
This file should still be a top-level script, so placed in the `pylhc` directory with `bsrt_analysis` and others

Reply:
better?

Reply:
Yes!